server/src/main/scala/com/fortysevendeg/smarthome/server/app/ServerBoot.scala

```diff
@@ -1,22 +1,41 @@
 package com.fortysevendeg.smarthome.server.app
 
 import cats.effect._
+import cats.effect.ExitCode
 import cats.syntax.flatMap._
 import cats.syntax.functor._
 import com.fortysevendeg.smarthome.config.ConfigService
 import com.fortysevendeg.smarthome.server.common._
+import com.permutive.pubsub.producer.{Model, PubsubProducer}
+import com.permutive.pubsub.producer.encoder.MessageEncoder
+import com.permutive.pubsub.producer.grpc.{GooglePubsubProducer, PubsubProducerConfig}
 import io.chrisdavenport.log4cats.Logger
 import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
 import pureconfig.generic.auto._
 
+import scala.concurrent.duration._
+
+
 abstract class ServerBoot[F[_]: ConcurrentEffect] {
 
+  implicit val encoder: MessageEncoder[Row] = (a: Row) => Right(a.toString.getBytes)
```

Comment:
This is not completely valid, we should serialize it properly. For the sake of simplicity, let's use `circe` and `json`, for example.